base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -93,6 +93,10 @@ where TBackend: TransactionBackend
     >,
     event_publisher: Publisher<TransactionEvent>,
     node_identity: Arc<NodeIdentity>,
+    callback_received_transaction: Option<unsafe extern "C" fn(*mut InboundTransaction)>,
+    callback_received_transaction_reply: Option<unsafe extern "C" fn(*mut CompletedTransaction)>,
+    callback_received_mine_completed_transaction: Option<unsafe extern "C" fn(*mut CompletedTransaction)>,
+    callback_received_detect_inbound_transaction_broadcast: Option<unsafe extern "C" fn(*mut CompletedTransaction)>,
```

Comment:
update as per above comment

Reply:
Percolate the rename throughout PR

Reply:
Updated throughout the pr in next commit